server/dmain.cpp

```diff
@@ -37,8 +38,7 @@ int wmain(int argc, wchar_t* argv[], wchar_t* envp[]) {
     std::vector<std::string> args;
     for (int i = 0; i < argc; ++i) {
         std::wstring argi16(argv[i]);
-        std::string argi8;
-        utf8::utf16to8(argi16.begin(), argi16.end(), std::back_inserter(argi8));
+        std::string argi8 = std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>, wchar_t>{}.to_bytes(argi16);
```

Comment:
`utf8::utf16to8()` is a lot nicer to use than `std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>, wchar_t>{}.to_bytes()` but I suppose in limited scope it's fine...